src/bin/pg_autoctl/debian.c

```diff
@@ -273,6 +265,14 @@ debian_init_postgres_config_files(PostgresSetup *pgSetup,
 				return false;
 			}
 		}
+		default:
+		{
```

Comment:
I agree and I looked a bit more into the reason I had to change this to fix the warnings. It turns out it's not because of the enum not matching fully, but because of the function not always having the right return type. i.e. `-Wreturn-type` instead of `-Wswitch`.

I now added an extra return to the function at the end. From what I can tell this is needed because in theory you could cast any number to the enum, so in those cases you would reach undefined behaviour when reaching the end of the function. 
By fixing it like this we will get a `-Wswitch` warning when there's an enum variant added, but the `-Wreturn-type` warning is silenced.